sandcats/public/privacy.html

```diff
@@ -0,0 +1,250 @@
+<!DOCTYPE html>
+<html>
+  <head>
+    <title>Sandstorm Sandcats: Privacy Policy</title>
+    <meta charset="utf-8">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <script type="text/javascript">
+      function showTerms() {
+        document.body.className = "terms";
+        if (history.replaceState) {
+          history.replaceState({}, "Sandstorm Sandcats: Terms of Service", "/terms.html");
+        } else {
+          document.title = "Sandstorm Sandcats: Terms of Service";
+        }
+      }
+      function showPrivacy() {
+        document.body.className = "privacy";
+        if (history.replaceState) {
+          history.replaceState({}, "Sandstorm Sandcats: Privacy Policy", "/privacy.html");
+        } else {
+          document.title = "Sandstorm Sandcats: Privacy Policy";
+        }
+      }
+    </script>
+    <style type="text/css">
+body {
+  font-family: sans-serif;
+  background-color: #eee;
+  padding: 0;
+  margin: 0;
+}
+
+section > * {
+  margin: 1em auto;
+  max-width: 640px;
+  box-sizing: border-box;
+}
+
+section > blockquote {
+  padding: 0 32px;
+  background-color: #eee;
+  overflow: hidden;
+}
+
+h1 {
+  font-size: 25pt;
+}
+
+h3 {
+  font-size: inherit;
+}
+
+#terms, #privacy {
+  position: absolute;
+  background-color: white;
+  margin: 64px 0 0;
+  border-top: 1px solid #888;
+  width: 100%;
+}
+
+#terms::before, #privacy::before {
+  display: block;
+  position: absolute;
+  top: -32px;
+  height: 15px;
+  border: 1px solid #888;
+  border-bottom: none;
+  background-color: white;
+  padding: 8px 0;
+  text-align: center;
+  width: 190px;
+}
+
+#terms::before {
+  content: "Terms of Service";
+  left: calc(50% - 320px);
+}
+
+#privacy::before {
+  content: "Privacy Policy";
+  left: calc(50% - 120px);
+}
+
+body.terms #terms, body.privacy #privacy {
+  z-index: 1;
+}
+
+body.privacy #terms > *, body.terms #privacy > * {
+  display: none;
+}
+
+@media screen and (max-width: 650px) {
+  section > * {
+    width: 100%;
+    padding-left: 8px;
+    padding-right: 8px;
+  }
+  #terms::before, #privacy::before {
+    width: calc(50% - 16px);
+  }
+  #terms::before { left: 8px; }
+  #privacy::before { right: 8px; left: auto; }
+}
+
+    </style>
+  </head>
+  <body class="privacy">
+    <section id="overview">
+<h1>Overview: Sandstorm Sandcats</h1>
+
+<p>Sandstorm is a platform for server-oriented and web-based applications. Sandstorm the platform is open source; anyone can run Sandstorm on their own personal server hardware. When you run Sandstorm on your own hardware, you can do anything you want with it: Sandstorm the platform is licensed under the Apache 2.0 open source license which places no restrictions on usage.</p>
```

Comment:
Ditto the comment below re: "Sandstorm the company".  If you really like that way of saying things, hyphenate it to indicate that it's one noun.